### ISSUE TITLE
Add support for Compasses

### ISSUE THREAD

**Issue by obeseocean** [SnosMe/awakened-poe-trade#560]
When doing a quick price check the ilvl is now automatically checked. For whatever reason this causes charged compasses to not be searchable - will show 0 results until the ilvl is unchecked causing extra steps per price check

**Comment by Hierofantus** [comment]
Can confirm what @obeseocean is reporting. APT inputs the item level of charged compases which results in 0 search results showing up.

**Comment by SnosMe** [comment]
There is no special support for them yet, they are currently handled as an Item of Normal rarity which as you can see makes weird selection of filters.

**Comment by Irfy** [comment]
Isn't this everything that's needed to fix this?

```
diff --git a/renderer/src/web/price-check/filters/create-item-filters.ts b/renderer/src/web/price-check/filters/create-item-filters.ts
index 90d76aa..ec720a8 100644
--- a/renderer/src/web/price-check/filters/create-item-filters.ts
+++ b/renderer/src/web/price-check/filters/create-item-filters.ts
@@ -63,8 +63,7 @@ export function createFilters (
   }
   if (
     item.category === ItemCategory.DivinationCard ||
-    item.category === ItemCategory.Currency ||
-    item.info.refName === 'Charged Compass'
+    item.category === ItemCategory.Currency
   ) {
     filters.searchExact = {
       baseType: item.info.name
@@ -77,6 +76,17 @@ export function createFilters (
     }
     return filters
   }
+  if (
+    item.info.refName === 'Charged Compass'
+  ) {
+    filters.searchExact = {
+      baseType: item.info.name
+    }
+    filters.itemLevel = {
+      disabled: true
+    }
+    return filters
+  }

   if (item.category === ItemCategory.Map) {
     if (item.rarity === ItemRarity.Unique && item.info.unique) {

```
I don't have a running node.js setup so can't really test. Let me know if this looks right and I'll make an untested PR.

**Comment by SnosMe** [comment]
@Irfy it's already fixed, just not released 4575ee5076a02e3837ddc5af15a31180521d47f0

**Comment by Irfy** [comment]
Thanks... I can't read -- It clearly says `@SnosMe SnosMe closed this in SnosMe/awakened-poe-trade@4575ee5 26 days ago`, I missed that. Oh well, the Node.js rabbit hole was fun nonetheless :)
